x-pack/plugins/aiops/public/components/spike_analysis_table/spike_analysis_table_groups.tsx

```diff
@@ -269,19 +273,33 @@ export const SpikeAnalysisGroupsTable: FC<SpikeAnalysisTableProps> = ({
             );
           }
         }
-        if (Object.keys(repeatedValues).length > 0) {
+        if (Object.keys(repeatedValues).length > 0 || hasExtraBadges) {
           valuesBadges.push(
             <>
               <EuiBadge
                 key={`$more-id`}
                 data-test-subj="aiopsSpikeAnalysisGroupsTableColumnGroupBadge"
                 color="hollow"
               >
-                +{Object.keys(repeatedValues).length}{' '}
-                <FormattedMessage
-                  id="xpack.aiops.explainLogRateSpikes.spikeAnalysisTableGroups.moreLabel"
-                  defaultMessage="more field/value pairs also appearing in other groups"
-                />
+                {hasExtraBadges ? (
```

Comment:
Both of these messages should handle single / plural, for the case where there is only one more pair:

<img width="631" alt="image" src="https://user-images.githubusercontent.com/7405507/202696280-21636798-82a4-4bf9-bc1d-0de114bc6d72.png">

Reply:
Good catch! Updated in bb9f06ecb662d5676d1c274d5577c04c33567895